src/main/java/Duke.java

```diff
@@ -75,9 +74,27 @@ private static int addEventTask(String userInput, Task[] tasks, int count) {
         String dateAndTime = details[1];
         tasks[count] = new Event(deadlineTask, dateAndTime);
         count++;
+        printNewTask(tasks, count);
+        return count;
+    }
+
+    private static void markTaskAsDone(Task[] tasks, String userInput) {
+        int itemNumber = Integer.parseInt((userInput.substring(5)));
+        tasks[itemNumber-1].markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("  " + tasks[itemNumber-1]);
+    }
+
+    private static void printNewTask(Task[] tasks, int count) {
         System.out.println("New task added: ");
         System.out.println(" " + tasks[count-1]);
         System.out.println("Now you have " + count + " tasks in the list.");
-        return count;
+    }
+
+    private static void listTasks(Task[] tasks, int count) {
+        System.out.println("Here are your tasks:");
+        for (int j = 0; j<count; j++) {
+            System.out.println(j+1 + ". " + tasks[j]);
+        }
```

Comment:
In terms of coding standard, it would be better if you can consider to keep space consistency, i.e. for (int j = 0; j < count; j++)...

Reply:
Well noted! 👍